setup.json

```diff
@@ -11,10 +11,11 @@
         "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
+        "Framework :: AiiDA",
         "Development Status :: 4 - Beta"
```

Comment:
maybe upgrade?

Reply:
This is still in development state. I need to update the documentation first